test/users/bguarraci/for_var_ref.future

```diff
@@ -0,0 +1,3 @@
+bug: although it makes sense to not pass in an iterator var as a ref, the error message doesn't make much sense.
```

Comment:
Since you want this to have a better error message, the .future file should start with

error message: